### PR TITLE
fix: disable WebRTC by default and add how to support it + websocket-star to the FAQ

### PR DESCRIPTION
**Pull request by diasdavid**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -344,7 +369,7 @@ const node = new IPFS({
       Swarm: [
         "/ip4/0.0.0.0/tcp/4002",
         "/ip4/127.0.0.1/tcp/4003/ws",
-        "/dns4/star-signal.cloud.ipfs.team/wss/p2p-webrtc-star"
+        "/dns4/wrtc-star.discovery.libp2p.io/wss/p2p-webrtc-star"
```
Comment (diasdavid) [inline README.md]: Waiting on https://github.com/libp2p/js-libp2p-webrtc-star/issues/126

File: README.md [hunk 2]
```diff
@@ -398,6 +417,33 @@ const node = new IPFS({
...
+#### Is there a more stable alternative to websocket-star that offers a similar functionality?
+
+Yes, websocket-star! A WebSockets based transport that uses a Relay to route the messages. To enable it, just do:
+
+```JavaScript
+const node = new IPFS({
+  config: {
+    Addresses: {
+      Swarm: [
+        '/dns4/ws-star.discovery.libp2p.io/wss/p2p-websocket-star'
```
Comment (diasdavid) [inline README.md]: Waiting on https://github.com/libp2p/js-libp2p-websocket-star-rendezvous/issues/9

File: README.md [hunk 3]
```diff
@@ -398,6 +417,33 @@ const node = new IPFS({
 
 The code above assumes you are running a local `signaling server` on port `9090`. Provide the correct values accordingly.
 
+#### Is there a more stable alternative to websocket-star that offers a similar functionality?
```
Comment (patrickdet) [inline README.md]: don't you mean `webrtc-star`?

Comment (diasdavid) [reply README.md]: good catch, thank you :)

**Comment by diasdavid** [comment]
image


Validated that websocket-star is indeed working with the newly deployed rendezvous point by @VictorBjelkholm, great job!

**Pull request merged**